modules/tinymce/src/core/main/ts/fmt/FormatUtils.ts

```diff
@@ -259,10 +259,10 @@ const isBlockFormat = (format: Format): format is BlockFormat =>
   Obj.hasNonNullableKey(format as any, 'block');
 
 const isWrappingBlockFormat = (format: Format): format is BlockFormat =>
-  isBlockFormat(format) && format.wrapper;
+  isBlockFormat(format) && format.wrapper === true;
 
 const isNonWrappingBlockFormat = (format: Format): format is BlockFormat =>
-  isBlockFormat(format) && !format.wrapper;
+  isBlockFormat(format) && format.wrapper !== true;
```

Comment:
```suggestion
  isBlockFormat(format) && !!format.wrapper;
```
or do we not use that style 🤔

Reply:
I think we try to avoid that style if possible.

Reply:
44 instances of it across the codebase 🤷‍♂️ 😂